<launch>
  <node name="remove_people_from_pcl" pkg="crowdbot_active_slam"
        type="remove_people_from_pcl" output="screen">
    <param name="pcl_callback_topic" value="/camera/depth/color/points"/>
    <param name="obstacle_callback_topic" value="/static_scan_extractor/dyn_obstacles"/>
    <param name="unknown_callback_topic" value="/static_scan_extractor/unknown_objects"/>
    <param name="max_dist_from_robot" value="3.5"/>
  </node>
</launch>
